### PR TITLE
io.hokus.Hokus

### PR DESCRIPTION
**Pull request by mildred**
Hokus is an electron app that help edit static websites locally.

See https://github.com/julianoappelklein/hokus/issues/17 and https://www.hokus.io. Please provide access to @julianoappelklein

### CODE REVIEW

File: io.hokus.Hokus.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "command": "run.sh",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
```
Comment (bilelmoussaoui) [inline io.hokus.Hokus.json]: Those cflags & cxxflags are not needed anymore on the FreeDesktop runtime 18.08

**Comment by bilelmoussaoui** [comment]
The app lacks an AppData file https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html
that file should be named `io.hokus.Hokus.appdata.xml` and installed under  `$PREFIX/share/metainfo`

A desktop file: https://standards.freedesktop.org/desktop-entry-spec/latest/
The file should be named `io.hokus.Hokus.desktop` and installed under `$PREFIX/share/applications`

Some icons could use those here https://github.com/julianoappelklein/hokus/tree/master/public and should be installed in`$PREFIX/share/icons/hicolor/$size$x$size$/apps` for all the sizes you have there and the icon must be named `io.hokus.Hokus` (which is the app-id, see the appdata/desktop specs for more info)

Those files should ideally live upstream (desktop & appdata file). The desktop file won't need any updates at all, but the appdata one needs updates before each release to add changelog (see releases tag on AppData specs).

The $PREFIX for flatpak is /app, those files could be used by other package management and they are standards that the app itself should follow and use and not Flatpak/Flathub specifics.

File: io.hokus.Hokus.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "io.hokus.Hokus",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
```
Comment (TingPing) [inline io.hokus.Hokus.json]:
```suggestion
     "base-version": "18.08",
```

**Comment by barthalion** [comment]
bot, build io.hokus.Hokus
testbot, build io.hokus.Hokus

**Comment by barthalion** [comment]
Hi @mildred, any updates on changes requested by @bilelmoussaoui and @TingPing?

**Pull request closed without merging**